### DISCUSSION TITLE
✅ Creating an OSPO Local Meetup - China (Chinese speaking)

### DISCUSSION THREAD

**Discussion by zhiqiang-yu** [todogroup/ospology#436] [Community Proposals]
I'm writing to apply for creating an OSPO Local Meetup -China  (Chinese speaking). In 2021, with help of LF APAC Office (HongKong) staff and Open Source Evangelist team, we have build an OSPO SIG in China. LF APAC OSPO SIG (China) had a series of local activities in last 3 years, like OSPO “10,000 Miles Journey”, which was focusing on meeting Linux Foundation member companies' representatives.

In 2023, we have launched the OSPO Summit (China) by community members and supported OSPOcon in GOTC and OSPO bof in KubeCon+OSS China. In the end of 2023, we went to Tokyo for Open Source Summit Japan and joined Local Meetup - Japan. It's so wonderful. We would like to create a similar OSPO Local Meetup in China community.

Milestones for the China OSPO Local Meetup would be:

- Start and grow OSPOlogyLive China and OSPO Local Meetups
- Build bridges between LF APAC OSPO SIG and the wider TODO Community
- Co-create open source content with regional and global OSPO practitioners
- Create new or adapt existing OSPO resources for Chinese organizations
- Bring discussions about the status of OSPOs in China to the TODO community
- Add previous OSPO “10,000 Miles Journey” into TODO community as local event
- Help local companies to climb the "Four Stages of the OSPO"
- Promote local companies to follow open source compliance policies
- Promote open collaboration with Japan, Korean local communities 

The organizers of this initiative and TODO Ambassadors are lijiangsheng1 and zhiqiang-yu

- [x] By checking this box, the organizers confirm that they have read the [ambassador guidelines](https://github.com/todogroup/ospology/blob/main/local-meetups/local-ambassadors-guidelines.md) and will take responsibility for the following tasks:

Managing registration and meetup descriptions through the Linux Foundation’s online community platform (content can also be mirrored on third-party platforms such as [meetup.com](http://meetup.com/)).
Providing the location and logistics for the meetup.
Ensuring the meetup adheres to the Linux Foundation's code of conduct.
Posting a summary of these meetings via to the [OSPO Forum](https://github.com/todogroup/ospology/discussions) in English each quarter

**Comment by anajsana** [comment]
Thank you @zhiqiang-yu for your submission! This has been shared to the SC for review and I will come back by the end of this week with resolution and next steps 🙏👏

**Reply by zhiqiang-yu** [reply]
Thanks and look forward to approval from the SC. Please also let the SC know our sincere request as following. If our proposal is approved, for the slack channel, we would like to have a chapter-china channel. Now there are 'chapter-japan' and 'chapter-apac'. We may leave 'chapter-apac' for future open collaboration of Japan community, China community, Korean one and etc. in APAC region.

**Comment by anajsana** [comment]
Thanks for your patience @zhiqiang-yu ! I am pleased to inform that the OSPO Local Meetup - China (Chinese speaking) proposal meet the criteria to create a new chapter /meetup and is now approved.
 
Also, @zhiqiang-yu and @lijiangsheng1 meet the criteria to become TODO ambassadors, given their long term commitment in advocating for TODO and OSPOs in china and encouraging contributions in Chinese to the OSPOlogy Repo, the TODO website, and the OSPO Book so this is also approved.

Below are the to-do list that the TODO PM (@anajsana) and new TODO Ambassadors must complete to finalize the onboarding of this new chapter and TODO Ambassadors:

- [x] @anajsana creates a new slack channel in TODO slack called # chapter-china
- [x] @zhiqiang-yu and @lijiangsheng1 to cross-promote via WeChat OSPO SIG Group
- [x] @anajsana creates a new chapter in the LF Online Community Platform named OSPO Local Meetup - China (zh-CN Speaking) and gives admin access to @zhiqiang-yu and @lijiangsheng1 to make edits to the landing page and create new meetups / gatherings or edit existing ones
- [x] @anajsana designs the OSPOChi China Mainland Mascot aligned with other regional OSPOChi mascots designs (e.g see japan OSPOChi Mascot here: https://community.linuxfoundation.org/ospo-local-meetup-japan-japanese-speaking/)
- [ ] @anajsana announces the creation of this new chapter via TODO and LF communication channels
- [x] @anajsana adds TODO ambassadors name in the Ambassadors page: https://todogroup.org/community/ambassadors/
- [ ] @zhiqiang-yu and @lijiangsheng1 to provide the SVG logo to add LF APAC to the OSPO Landscape as OSPO Associates: https://landscape.todogroup.org/

**Reply by zhiqiang-yu** [reply]
For the SVG logo of LF APAC OSPO SIG, LF APAC management team suggested we create a completely a new logo, not using LF APAC logo. So we may need some time for it.

**Comment by anajsana** [comment]
@zhiqiang-yu the chapter has been successfully created: https://community.linuxfoundation.org/ospo-local-meetup-china-zh-cn-speaking/ (please change the Chinese message on the description as desired, it was an initial template created with automatic translation) I would need the email you use to log in to the LF platform to add you as chapter admin, so you can create new events in the chapter and manage registrations 🙆‍♀️  (same for @lijiangsheng1)

OSPOChi mascot for the China Chapter is also live! 
![ospochilogo-china](https://github.com/todogroup/ospology/assets/43671777/7760ddd7-bb92-46e2-a769-ef8c42a7aca6)

**Reply by anajsana** [reply]
you can send me the email via private slack message or email 🙂

**Reply by zhiqiang-yu** [reply]
Fine, I‘ve sent you in slack message

**Reply by lijiangsheng1** [reply]
Hi, @anajsana  unfortunately， we can't use China Flag in public without government permission. so we need change other picture to replace.  Panda or KungFu is ok.

**Reply by anajsana** [reply]
Oh, I see! I've created three more versions. One with a panda and the other with the zh-CN. Which one do you prefer? @zhiqiang-yu @lijiangsheng1 ?

Option A: 
![Option A](https://github.com/todogroup/ospology/assets/43671777/74d7f043-149d-4860-94ca-0879f721efce)
Option B:
![2](https://github.com/todogroup/ospology/assets/43671777/cd10baac-c60d-4c5f-aff0-9297aae01167)

Option C:

![ospochilogo-china(1)](https://github.com/todogroup/ospology/assets/43671777/92805b55-ee05-4b77-a0bc-4e0d7f55f522)

**Reply by zhiqiang-yu** [reply]
Wonderful, I prefer option B
